packages/graphql/mixin.server.js

```diff
@@ -21,12 +21,19 @@ class GraphQLMixin extends Mixin {
     super(config, element);
 
     this.client = this.createClient(options);
-    this.shouldPrefetchOnServer = undefined;
+    this.prefetchOnServer =
+      typeof this.config.prefetchOnServer === 'boolean'
+        ? this.config.prefetchOnServer
+        : true;
   }
 
   bootstrap(request, response) {
-    if (response && response.locals) {
-      this.shouldPrefetchOnServer = response.locals.shouldPrefetchOnServer;
+    if (
+      response &&
+      response.locals &&
+      typeof response.locals.prefetchOnServer === 'boolean'
+    ) {
+      this.prefetchOnServer = response.locals.prefetchOnServer;
```

Comment:
Only when `response.locals.prefetchOnServer` is actually set will we override the current behavior from the config.

Reply:
I suppose we should also add `this.prefetchOnServer !== false`, so that users actually **can** disable prefetching via config.